scimma_admin/scimma_admin/settings.py

```diff
@@ -42,8 +42,24 @@ def get_localdev_secret(name):
     print(cp.sections())
     return cp["secrets"][name]
 
+SCIMMA_ENVIRONMENT = os.environ.get("SCIMMA_ENVIRONMENT", default="local")
 
-PRODUCTION = os.getenv("SCIMMA_ADMIN_PROD") is not None
+_aws_name_prefixes = {
+    "local": None, # AWS variables are not used for local testing
+    "dev": "", # this is empty for historical reasons, it should probably be renamed in future
+    "demo": "demo-",
+    "prod": "prod-",
+}
+
+if not SCIMMA_ENVIRONMENT in _aws_name_prefixes.keys():
+    raise RuntimeError(f"Specified environment (${SCIMMA_ENVIRONMENT}) is not known")
```

Comment:
The dollar sign is extraneous here; this will generate a string like `Specified environment ($foobar) is not known`.

Reply:
Oops, yes. Writing too many shell scripts lately. . .